### PR TITLE
Add Shell scripts for simulation

### PR DESCRIPTION
**Pull request by xadahiya**
- Init script add
- Update .gitignore
- Update requirements.txt
- Add instructions in README.md

### CODE REVIEW

File: bootstrap-dev.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+git clone http://github.com/xadahiya/hydra-flock-gui.git sim_gui
+
+# Drone1
+git clone http://github.com/xadahiya/hydra-flock-drone.git sim_drone1
+# Drone2
+git clone http://github.com/xadahiya/hydra-flock-drone.git sim_drone2
+# Drone3
+git clone http://github.com/xadahiya/hydra-flock-drone.git sim_drone3
+# Drone4
+git clone http://github.com/xadahiya/hydra-flock-drone.git sim_drone4
```
Comment (chrizandr) [inline bootstrap-dev.sh]: All URLs need to be HTTPS not HTTP. Please change them.

**Comment by chrizandr** [comment]
```
  File "/home/chrizandr/hydra-flock/hydra-flock-demo/sim_drone3/flock_drone/mechanics/distance.py", line 1, in <module>
    from haversine import haversine
ImportError: No module named 'haversine'

```
Also getting these errors

**Comment by xadahiya** [comment]
@chrizandr Forgot to add it to requirements, please run `pip install haversine` for now

File: init.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
+echo "Starting Simulation Central Controller"
+python sim_controller/flock_controller/main.py &
+echo "Central Controller started successfully!"
+
+## Wait for 2 seconds.
+/bin/sleep 4
```
Comment (chrizandr) [inline init.sh]: Make all sleep to 8 seconds, there may be synchronisation errors if the values are small.

**Pull request merged**